db-connection.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
I don't understand how git can still see this when it is in gitignore.  Might be worth asking Charlie about it.